.github/comment.yml

```diff
@@ -1,60 +1,4 @@
 ---
-- rule:
```

Comment:
Redundant because this is now included in.....

Reply:
Some of the text is in PR description templates (I also touched them in this PR),
some of the text is in [requiredLabelRules](https://devdiv.visualstudio.com/DevDiv/_git/openapi-alps?path=/private/openapi-kebab/src/bots/pipeline/pipelineEventListener/requiredLabelsRules.ts&version=GBmain&line=52&lineEnd=106&lineStartColumn=1&lineEndColumn=2&lineStyle=plain&_a=contents) and some of it in [checksWorkflowInfo](https://devdiv.visualstudio.com/DevDiv/_git/openapi-alps?path=/private/openapi-kebab/src/bots/pipeline/pipelineEventListener/checksWorkflowInfo.ts&version=GBmain&line=69&lineEnd=88&lineStartColumn=1&lineEndColumn=132&lineStyle=plain&_a=contents)

I do expect to iterate over next few weeks on the text to make it better.